helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -130,6 +130,39 @@ controller:
     # The name tha the RBAC role will get.
     name: nginx-ingress-role
 
+  # controller.livenessProbe
+  # Liveness probe configuration
+  # Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  livenessProbe:
+    enabled: true
+    failureThreshold: 30
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 5
+
+  # controller.readinessProbe
+  # Readiness probe configuration
+  # Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  readinessProbe:
+    enabled: true
+    failureThreshold: 3
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 1
+
+  # controller.startupProbe
+  # Startup probe configuration
+  # Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  startupProbe:
+    enabled: false
```

Comment:
startupProbe is disabled by default

Reply:
did you check the number from upstream or measure the starting time of Nginx?

Reply:
Startup probe is left on probe defaults. Decided not to invest much time in tuning it. Focused on liveness probe, making it more resilient than readiness probe. Was satisfied with just having startup probe as an option. It may require tuning if enabled.